pkg/kubelet/pleg/generic.go

```diff
@@ -475,7 +475,9 @@ func (g *GenericPLEG) updateCache(ctx context.Context, pod *kubecontainer.Pod, p
 	// Evented PLEG after the event has been received by the Kubelet.
 	// For more details refer to:
 	// https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/3386-kubelet-evented-pleg#timestamp-of-the-pod-status
-	if utilfeature.DefaultFeatureGate.Enabled(features.EventedPLEG) && isEventedPLEGInUse() && status != nil {
+	// Regardless of whether EventedPLEG is enabled or not, status.TimeStamp will be calculated by
+	// runtime.GetPodStatus() as long as it does return none-null PodStatus.
+	if status != nil {
```

Comment:
This seems reasonable, but you should not make changes here, as it may break the existing behavior of the Generic PLEG. You should revert this commit and narrow the scope of this PR to only modifying the timestamp.

If you still feel that this change is necessary, please open a new PR for review.